pkg/skaffold/constants/constants.go

```diff
@@ -64,6 +64,9 @@ const (
 
 	DefaultRPCPort     = 50051
 	DefaultRPCHTTPPort = 50052
+
+	PodResourceType     = "pod"
```

Comment:
maybe make it into a constant/enum pattern? 

```go
type ResourceType string
const (
  Pod ResourceType = "pod"
  Service ResourceType = "service"
) 
```

Reply:
done!